im3components/wrf_tell/wrf_tell_counties.py

```diff
@@ -0,0 +1,204 @@
+import argparse
```

Comment:
See import grouping and ordering via PEP 8 here:  https://www.python.org/dev/peps/pep-0008/#imports